### ISSUE TITLE
Add Contributors

### ISSUE THREAD

**Issue by martinthomson** [quicwg/base-drafts#3278]
The list of people who have made contributions is LONG.  This issue will track creation of a list of contributors.

This will list people's name properly.  We're now allowed to use non-ASCII codepoints in names, we should exploit that fully.  Names that are mostly Latin can use the native form; for people with non-Latin names (like @kazuho), we probably should include an anglicized version so that they remain recognizable to those of us not sufficiently versed in their native language.

My current inclination is to avoid listing affiliations for a range of reasons, including the number of affiliation changes over the course of this process and the fact that this information was not always available.

This isn't an open invitation to get your name listed.  Nor is this an opportunity to get your paper cited.  My view is that editors reserve the right to decide who is listed and that is not open to debate.  I also hold that non-substantive contributions like fixing a typo - while greatly appreciated - does not in itself justify inclusion.  All that said, we're probably likely to miss at least some of the many people who have helped out.  Once the list is assembled, let the editors know if there are omissions or corrections.

**Comment by larseggert** [comment]
Do we need this? IMO it'll just open up a can of worms when someone later on feels like they should be on this list but aren't.

**Comment by martinthomson** [comment]
Consider the can opened.

**Comment by ekr** [comment]
FWIW, we did this with TLS 1.3, and didn't have any serious problems. At
the end, I asked people who felt i had missed them to speak up and there
wasn't any controversy.

**Comment by DavidSchinazi** [comment]
I think this is a very good idea. Thanks for doing the work @martinthomson.

**Comment by janaiyengar** [comment]
@larseggert : I think we can make this work.

**Comment by MikeBishop** [comment]
FWIW, my plan was to use `git blame -w draft-ietf-quic-http.md | perl -n -e '/^.*?\((.*?)\s+[\d]{4}/; print $1,"\n"' | sort -f | uniq -c | sort -n` to generate an initial list and then use judgement where to draw the cut-off and/or add people who contributed on the mailing list but not on git.

**Comment by martinthomson** [comment]
That's part of what I did.  `git shortlog -sne $file` might be better than blame, especially since that captures changes that were subsequently erased by other commits.  There are some duplicates as some people change names, and `root` appears to have contributed in at least one case, but it works well enough.

I also walked through the issues and mailing list to see if there were others who made contributions other than commits.
